client/me/purchases/payment/edit-card-details/index.jsx

```diff
@@ -72,7 +72,8 @@ class EditCardDetails extends PurchaseCardDetails {
 					apiParams={ this.getApiParams() }
 					initialValues={ this.props.card }
 					recordFormSubmitEvent={ this.recordFormSubmitEvent }
-					successCallback={ this.successCallback } />
+					successCallback={ this.successCallback }
+					actionType="card_update" />
```

Comment:
I have one tiny stylistic request. Can we use this coding style:
```js
actionType="card_update"
```
in all other places? :)